manifests/kustomize/base/cache/cache-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cache-server
+  labels:
+    app: cache-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cache-server
+  template:
+    metadata:
+      labels:
+        app: cache-server
+    spec:
+      containers:
+      - name: server
+        image: gcr.io/ml-pipeline/cache-server:test
```

Comment:
nit - in the future, please do not push test image to ml-pipeline, the production gcr.  you could add any placeholder here before add this component as part of the pipeline deployment.